packages/react/.eslintrc.js

```diff
@@ -23,6 +23,7 @@ const rules = {
       varsIgnorePattern: 'React|Cds',
     },
   ],
+  'import/extensions': ['error', 'ignorePackages'],
```

Comment:
I only added this to the react package - should it be across the whole repo? Scott left a comment on my PR yesterday about file extensions so I'm thinking yes?

Reply:
I thought we had this turned on in Core but it doesn't look like that's the case. We should definitely add this rule to the Core eslint config.

Reply:
Angular too? If I add it at the root there's only 2 or 3 issues in core, but a lot more in angular.

Reply:
Maybe just start with Core. For Core it can make or break the library with CDN or ESM tools. Angular not as much because its always using a build step somewhere.